### PR TITLE
STM32U5

### PR DESCRIPTION
**Pull request by rleh**

### CODE REVIEW

File: tools/generator/dfg/stm32/stm.py [hunk 1]
```diff
@@ -6,6 +6,18 @@
...
+[
+    "STM32U59",
+    "STM32U5A",
+]
+
+def ignoreDevice(device_id: str) -> bool:
+    for ignore in ignored_devices:
+        if device_id.startswith(ignore):
+            return True
+    return False
```
Comment (rleh) [inline tools/generator/dfg/stm32/stm.py:19]: Not sure if that is the best way to ignore some devices (here: STM32U5{95, 99, A5, A9}) that are not published (yet) and thus are missing datasheet and relevant information.

I my opinion I it not useful to generate device files for this devices so we should somehow ignore them.

**Comment by chris-durand** [comment]
Looks like ST added yet another type of a completely different DMA implementation ...

**Review by chris-durand — APPROVED** [review]
Thanks!

**Comment by rleh** [comment]
> Looks like ST added yet another type of a completely different DMA implementation ...

Yes. I ignored DMA completely for now in order to get STM32U5 support merged soon and not to create another PR that will get marked as _stale_ in a few weeks.

File: tools/generator/dfg/stm32/stm.py (hunk 1 above)
Comment (salkinium) [reply tools/generator/dfg/stm32/stm.py:19]: Meh, doesn't look like we have a choice if the documents are not there. We could guess the memory layout, but then it'll just be a problem later.

**Review by salkinium — APPROVED** [review]

**Pull request merged**